proxy/proxy.go

```diff
@@ -871,7 +871,14 @@ func (p *Proxy) makeBackendRequest(ctx *context) (*http.Response, *proxyError) {
 		p.log.Errorf("could not map backend request, caused by: %v", err)
 		return nil, &proxyError{err: err}
 	}
-
+	if ctx.route.BackendType == eskip.LBBackend && ctx.route.LBEndpoints != nil {
+		// TODO: check req.URL vs req.Host
+		endpoint := ctx.route.LBEndpoints.Get(req.URL.Host)
+		endpoint.Metrics.IncInflightRequest()
+		defer func() {
```

Comment:
You can drop the wrapping func, just pretend function call with defer, if you need multi line defer it’s how you do it now

Reply:
Done.